### PR TITLE
Carousel Implementation

### PR DESCRIPTION
**Pull request by mpstanford**
@mbbph @vargas055 @dbooyah93

### CODE REVIEW

File: client/src/Image.jsx [hunk 1]
```diff
@@ -1,39 +1,25 @@
-import React from 'react';
-import axios from 'axios';
+import React, { useState } from "react";
```
Comment (dbooyah93) [inline client/src/Image.jsx:1]: Cool deconstruct

File: client/src/index.jsx [hunk 2]
```diff
@@ -1,7 +1,93 @@
...
+    axios.get(`/products/${productId}`)
+      .then((response) => {
+        this.setState({
+          images: response.data.imageUrls,
+        });
+      })
+      .catch((error) => {
+        console.log('There was an error: ', error);
+      });
+  }
```
Comment (dbooyah93) [inline client/src/index.jsx:38]: lines 27 - 38 look really nice and easy to read.

**Review by dbooyah93 — APPROVED** [review]
You committed some .png files.  I assume it was necessary and I am not sure what is best practice, but it was odd.

Seeing your API tests helped me.

Your code looks good. Merge it!

**Comment by mpstanford** [comment]
chrome-capture

Not sure if the above GIF will show up...

**Pull request merged**